src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1032,6 +1045,91 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: DataSource,
     }
   }
 
+  def listGroupMembers(groupName: String) = {
+    asAdmin {
+      dataSource.inFutureTransaction() { txn =>
+        Future {
+          containerDAO.authDAO.loadGroup(RawlsGroupRef(RawlsGroupName(groupName)), txn) match {
+            case None => RequestComplete(ErrorReport(StatusCodes.NotFound, s"Group ${groupName} does not exist"))
+            case Some(group) =>
+              val memberUsers = group.users.map(u => containerDAO.authDAO.loadUser(u, txn).get.userEmail.value)
+              val memberGroups = group.subGroups.map(g => containerDAO.authDAO.loadGroup(g, txn).get.groupEmail.value)
+              RequestComplete(StatusCodes.OK, (memberUsers ++ memberGroups).toSeq)
+          }
+        }
+      }
+    }
+  }
+
+  def createGroup(groupRef: RawlsGroupRef) = {
+    asAdmin {
+      dataSource.inFutureTransaction() { txn =>
+        Future {
+          containerDAO.authDAO.loadGroup(RawlsGroupRef(groupRef.groupName), txn) match {
+            case Some(_) => RequestComplete(ErrorReport(StatusCodes.Conflict, s"Group ${groupRef.groupName} already exists"))
+            case None =>
+              gcsDAO.createGoogleGroup(groupRef.groupName.value)
```

Comment:
gcsDAO calls already execute in Futures, so they should be composed with other Futures, not nested inside them.  It's possible that the `RequestComplete` will be returned to the caller before `createGoogleGroup` runs.  Also, the success value of the Google Group creation is not checked.

This comment generally applies to the other services as well.